assets/html/js/metadata.js

```diff
@@ -1,6 +1,6 @@
 // libraries: jquery
 // arguments: $
-
+/*
 $(document).ready(function () {
   let meta = $("div[data-docstringscollapsed]").data();
 
```

Comment:
My understanding based on what you just explained to me off-band is that this function dealt with scrolling the view port (which was meant to fix certain issues but never really worked well), and which is now unnecessary with your changes as the browser just does the right thing.

So I would suggest we just delete it!